standalone-metastore/src/main/thrift/hive_metastore.thrift

```diff
@@ -871,6 +871,7 @@ struct ReplTblWriteIdStateRequest {
     4: required string dbName,
     5: required string tableName,
     6: optional list<string> partNames,
+    7: optional string catName,
 }
 
```

Comment:
/ Valid Write ID list of one table wrt to current txn
struct TableValidWriteIds {
    1: required string fullTableName,  // Full table name of format <db_name>.<table_name>

shouldn't this be a cat.db.table?

Reply:
Agreed.